CHANGELOG.md

```diff
@@ -55,7 +55,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -
 -
 -
--
+- SARIMAX parameters fix ([#459](https://github.com/tinkoff-ai/etna/pull/459))
```

Comment:
looks like a merge problem?

Reply:
No, It's example how to properly use placeholders. We should fill them at random place to minimize probability of merge conflicts